backend/src/components/appConfig.js

```diff
@@ -168,7 +165,7 @@ const appConfig = {
           'Content-Type': 'application/json; charset=utf-8'
         }
       });
-      log.verbose('postAppConfig', utils.prettyStringify(webAdeResponse.data));
+      log.verbose('postAppConfig', JSON.stringify(webAdeResponse.data));
```

Comment:
I see that most of the calls to utils.prettyStringify have been changed to JSON.stringify.  Let's remove that method, update any of the remaining 3 calls to use JSON.stringify, and we can just pass in the indent there if required.  It's one fewer thing to test, one fewer proprietary call for readers to look up.

Reply:
Fair enough at this point - there isn't as much value for us to preserve this util function at this point as it's pretty simple with what it's doing to begin with.